src/redux/local-storage.js

```diff
@@ -24,6 +24,11 @@ export const getLocalStorageBoolean = (key, defValue) => {
     return value === 'true';
 };
 
+export const getLocalStorageInteger = (key, defValue) => {
```

Comment:
See jon comment on usefulness of this function 
https://github.com/gridsuite/gridstudy-app/pull/193#discussion_r490432841